### ISSUE TITLE
vcgUpdateNormals integer check

### ISSUE THREAD

**Issue by zerweck** [zarquon42b/Rvcg#3]
You check if the arguments are integers by using
```R
!prod(is.integer(pointcloud))
```
but that will be triggered by your own standard arguments, since R does not default to integer when giving a number without further context. 
See:
```R
> typeof(1)
[1] "double"
> typeof(1L)
[1] "integer"
```
So with values like c(30,1), the output will not change, and the function will always use c(10,0).

I would propose one of two ways to solve this: 
    1. Changing the default parameter to c(10L,0L) and state the requirement in the help file more explicit, plus a stop("Please state pointcloud parameters in explicit integers, i.e. c(10L,0L)") in the if TRUE statement.
    2. Changing the check to a forced coercion via as.integer and warn the users in the help file about it. 

Since RupdateNormals does not seem to throw an error, even though the standard submitted argument is of type double, the check may even not be necessary?

**Comment by zarquon42b** [comment]
Hi, this test is outdated, as I am catching exceptions on c++ level now. I will remove it.
Best
Stefan

**Comment by zarquon42b** [comment]
Fixed in d38de568c12964e366fb1b2103f949b8354eec95
